.github/CONTRIBUTING.md

```diff
@@ -114,15 +114,16 @@ Please try to add corresponding testcase to runner or unit.
 Documentation is stored in `/docs` directory in markdown format.
 
 **Documentation for helpers is a part of a source code**.
-Whenever you need to update docs for a helper do it inside a .js file.
 
-In order to generate new documentation from source code run the following command with [Robo](https://robo.li/):
+> **Whenever you need to update docs for a helper do it inside a .js file.**
+
+After you updated docblock in JS file, generate markdown files with next command:
 
 ```
-robo docs:helpers
```

Comment:
Maybe replace to `npm run docs` or something similar.

Reply:
there is no need in npm layer to do that.